docs/Project-Configuration/Project-and-Distro-Settings.md

```diff
@@ -30,7 +30,15 @@ read or write (or no) access to project settings, logs, task
 annotations, and more. These can be requested for specific members or
 for entire guilds. The project admins (or Evergreen admins, if no
 project admins exist) are then able to approve/reject requested access
-for their project.
+for their project. 
+
+In order to access this functionality, you must look up the the MANA resource 
+with the type `Evergreen Project` with the name of the project that
+permissions need to be requested for. Once the project is selected, click on 
+the `Request Access` button and fill out the `Requesting for` field to see
+the specific types of permissions that can be requested on Evergreen as pictured below.
```

Comment:
Non blocking but can we update the authorization section of the dev guide saying that these values map directly to our role types + permission levels: https://docs.google.com/document/d/1KxK2-JhKiHg7ydoEJN6rAf63k94KE_5-DqlYBj48pig/edit#heading=h.grkjk8um3vob

Reply:
added